src/component/categoryItem/category-item.js

```diff
@@ -12,7 +12,6 @@ class Category extends React.Component {
       categoryRemove,
```

Comment:
`CategoryItem` component should `mapStateToProps` for the expenses, then map over them (just like the `Dashboard` component maps over `categories`) and render an `<ExpenseItem />` component. This is how to get your expenses showing up.